addons/account_edi/models/account_journal.py

```diff
@@ -71,7 +80,8 @@ def _compute_edi_format_ids(self):
         else:
             protected_edi_formats_per_journal = defaultdict(set)
 
-        for journal in self:
+        for batch in batched(self, 10000):
```

Comment:
Like @jco-odoo said, this doesn't make any sense.
Having more than 100 journals would already be a lot. Having more than 1000 would be exceptional. Having more than 10000 is complete utter nonsense.

On our production, with 15 companies activated, we have 295 journals. (as far as I can see)

Reply:
Of course it's stupid. But as of today, this leads to a traceback.